ledger-api/grpc-definitions/com/daml/ledger/api/v1/transaction.proto

```diff
@@ -38,7 +38,7 @@ message TransactionTree {
 
   // The absolute offset. The format of this field is described in ``ledger_offset.proto``.
   // Required
-  string offset = 6;
+  int64 offset = 6;
```

Comment:
We propose switching to longs for all offsets exposed on the new endpoints.
This of course is not done for the old endpoints which are still there, and use stringual, and merged-domain offsets (with no strong causal guarantees).
 @meiersi-da @moritzkiefer-da can you confirm please, that this is ok?

Reply:
that is OK for us. Albeit: we might find after the PoC that we want to keep the string-based offsets to keep more of the Daml 2.0 API to simplify the life of the app authors while delivering quickly.

Reply:
As discussed with @meiersi-da : let's keep the Stringual offsets on the API for keeping it closer to 2.0 API